packages/widgetbook_generator/lib/src/generators/use_case_generator.dart

```diff
@@ -52,12 +57,15 @@ class UseCaseGenerator extends GeneratorForAnnotation<UseCase> {
     final useCasePath = await resolveElementPath(element, buildStep);
     final componentPath = await resolveElementPath(type.element!, buildStep);
 
+    final pathInWidgetbook = path ?? Tree.getPathParts(componentUri).join('/');
```

Comment:
I think moving `getPathParts` to `UseCaseGenerator` instead of `Tree` now makes sense, since this function is no longer used in the `Tree` class and is only used here.

You may as well adjust the **name** and the **return type** so you do not need to do `join` again here.